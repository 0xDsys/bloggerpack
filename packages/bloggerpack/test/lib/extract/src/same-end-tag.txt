<extract-start>
foo
</extract-end>

<nope>
bar
</extract-end>

<nope>
baz
</extract-end>
