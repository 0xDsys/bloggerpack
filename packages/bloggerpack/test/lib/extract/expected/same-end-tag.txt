/* src/same-end-tag.txt */

foo
